SphericalShell/OpenMC_Output_Processing/SS_Depletion_PostProcessing.py

```diff
@@ -0,0 +1,62 @@
+import openmc
+import openmc.deplete
+import numpy as np
+import matplotlib.pyplot as plt
+
+def extract_nuclides(dep_file_path, time_units, depletable_mat_index) : 

```

Comment:
It seems like the purpose of this is to collect a list of all the nuclides that appear, excluding the list of stable nuclides at beginning of life.  Is that right?  Do don't seem to actually record/return any data about those nuclides as a function of time.

Reply:
This function just stores all the new nuclides that were created (and any unstable ones at the beginning of operation), so that they can be easily retrieved later. The get_atoms method (used in plot_save_data) returns nuclide concentration as a function of time.